cmd/apps/grafana_app.go

```diff
@@ -56,6 +56,11 @@ func MakeInstallGrafana() *cobra.Command {
 			return err
 		}
 
+		err = addHelmRepo("stable", "https://kubernetes-charts.storage.googleapis.com", true)
+		if err != nil {
+			return err
```

Comment:
Are you sure this should return? What if it's already there and you run this twice in a row?

Reply:
I copied it from the helm Nginx app, as you said, but I tested it and you can add it multiple
times without an error. This should work.

Reply:
Perfect. Thank you